wolox-books/src/app/screens/auth/screens/book-list/book-list.component.html

```diff
@@ -1,3 +1,4 @@
+<app-navbar></app-navbar>
```

Comment:
Is necessary add navbar here?? you already put navbar component in the parent

Reply:
Sorry, fixed 😃